Source/Python/WarpX.cpp

```diff
@@ -46,6 +46,18 @@ void init_WarpX (py::module& m)
             //py::overload_cast< int >(&ImpactX::boxArray, py::const_),
             py::arg("lev")
         )
+        .def("multifab",
+            [](WarpX const & ix, std::string const multifab_name) {
```

Comment:
ooops, `ix` was from ImpactX :D we could call this argument `warpx` or `wx` or so.